src/amr/messengers/hybrid_hybrid_messenger_strategy.hpp

```diff
@@ -97,8 +116,41 @@ namespace amr
             : HybridMessengerStrategy<HybridModel>{stratName}
             , resourcesManager_{std::move(manager)}
             , firstLevel_{firstLevel}
+            , magneticInitRefiners_{resourcesManager_}
+            , electricInitRefiners_{resourcesManager_}
+            , magSharedNodesRefiners_{resourcesManager_}
+            , magGhostsRefiners_{resourcesManager_}
+            , magPatchGhostsRefiners_{resourcesManager_}
+            , elecSharedNodesRefiners_{resourcesManager_}
+            , elecGhostsRefiners_{resourcesManager_}
+            , currentSharedNodesRefiners_{resourcesManager_}
+            , currentGhostsRefiners_{resourcesManager_}
+            , rhoGhostsRefiners_{resourcesManager_}
+            , velGhostsRefiners_{resourcesManager_}
+            , domainParticlesRefiners_{resourcesManager_}
+            , interiorParticleRefineOp_{std::make_shared<InteriorParticleRefineOp>()}
```

Comment:
any member that does not require an input variable could be defaulted on declaration? 

e.g.

```
std::shared_ptr<RefineOperator> interiorParticleRefineOp_ = std::make_shared<InteriorParticleRefineOp>();
```

could make this constructor a bit more manageable